### DISCUSSION TITLE
PyMisp in Windows 10 - ImportError: attempted relative import with no known parent package

### DISCUSSION THREAD

**Discussion by cemarins** [MISP/PyMISP#722] [Q&A]
Good morning.

I've installed PyMisp on my Windows 10 machine to play a little with our MISP instance. Bellow I reproduce the directory structure regarding Python and PyMisp.

```
|-Python39 (directory)
   |-python.exe
   |-python3.dll
   | -Lib (directory)
   |-...
   |-pymisp (directory)
      |-api.py
      |-__init__.py
      |-exceptions.py
      |...
      |-data (directory)
      |-tools (directory)
      |-__pycache__

```
When I run python api.py from the command prompt I get the following error:

```
D:\_tools\Python39\pymisp>python api.py
Traceback (most recent call last):
  File "D:\_tools\Python39\pymisp\api.py", line 21, in <module>
    from . import __version__, everything_broken
ImportError: attempted relative import with no known parent package
```

I thought it would not get this error with that files and directories structure. What am I doing wrong?
Regards,
Carlos Eduardo.

**Comment by Rafiot** [comment]
Hi,

So, first of all, PyMISP has never been tested on Windows, so no promises it will ever work

But it's not the problem you're having here: you're trying to execute a library, it's not how you can use PyMISP. First of all, you need to install it (`pip3 install pymisp`), and then, I'd recommend you to look at the examples, and the documentation: https://pymisp.readthedocs.io/en/latest/
